internal/service/advancedclustertpf/resource_test.go

```diff
@@ -20,8 +20,26 @@ func TestAccAdvancedCluster_basic(t *testing.T) {
 }
 
 func configBasic() string {
-	return `	
+	return `
+	 /*	
+	 // TODO: resource not used yet until we have a fake model so test can run
 		resource "mongodbatlas_advanced_cluster" "test" {
```

Comment:
nice that the test can run!

Reply:
yes for the data sources, no quite yet for the resource because we need to populate at least the computed attributes with a "fake" model, this is the next PR I was thinking to do so a no-op resource (no calling Atlas) in test can be execeuted